assets/src/data/model/datetime/status-helper.js

```diff
@@ -78,13 +79,33 @@ export const isUpcoming = ( DateTimeEntity ) => {
 /**
  * @function
  * @param {Object} DateTimeEntity model object
- * @return {boolean} true if ticket is archived
+ * @return {boolean} true if date is archived
  */
 export const isTrashed = ( DateTimeEntity ) => {
 	assertDateTimeEntity( DateTimeEntity );
 	return DateTimeEntity.deleted;
 };
 
+/**
+ * @function
+ * @param {Object} DateTimeEntity model object
+ * @return {boolean} true if date is cancelled
+ */
+export const isCancelled = ( DateTimeEntity ) => {
+	assertDateTimeEntity( DateTimeEntity );
+	return DateTimeEntity.cancelled;
+};
+
+/**
+ * @function
+ * @param {Object} DateTimeEntity model object
+ * @return {boolean} true if date has been postponed
+ */
+export const isPostponed = ( DateTimeEntity ) => {
+	assertDateTimeEntity( DateTimeEntity );
+	return DateTimeEntity.postponed;
```

Comment:
wish i could blame this on drug use or alcohol or something but unfortunately I can't...
no idea why I thought that was a property or what I was thinking... 
it will just return `undefined`... but still...

¯\_(ツ)_/¯